subscriptions/templates/subscriptions/update.html

```diff
@@ -0,0 +1,29 @@
+{% extends "account/base.html" %}
+
+{% load static %}
+{% load i18n %}
+
+{% block title %}{% if stripe %}{% trans "Change plan" %}{% else %}{% trans "Add plan" %}{% endif %}{% endblock %}
+
+{% block content %}
+    {% include 'account/_logged_in_sidebar.html' %}
+    <section class="account-form">
+        <h2>{% if stripe %}{% trans "Change plan" %}{% else %}{% trans "Add plan" %}{% endif %}</h2>
+
+        <form id="signup_form" method="post" action="{% url "subscription_update" %}" autocapitalize="off">
```

Comment:
Filling in this form will remove the cancellation, as I understand it (whether you change plan or not), so it should probably say that somewhere, yep.